docs/content/en/docs/install/cert-manager.md

```diff
@@ -27,73 +27,45 @@ The steps are:
 
 * Install the cert-manager of your choice
   if it is not already installed.
-* Modify the `Deployment` manifest of each KLT operator component.
-* Add the `Certificate` CRD for the cert-manager you are using.
+* Add the `Certificate` and `Issuer` CRs for `cert-manager.io`.
+* (optional) Install Keptn without the built-in `klt-cert-manager` via Helm
```

Comment:
```suggestion
* (optional) Disable the built-in  `klt-cert-manager` before installing KLT.
   KLT then looks for the `cert-manager.io` `Certificate` and `Issuer` resources
   and uses those.
```

Reply:
The disabling of cert-manager happens during KLT installation. I would rather keep this sentence as it is

Reply:
I will defer to you because I don't completely understand this.  But if the cert-manager.io resources are present, what determines whether KLT co-exists with them or uses them instead of the built-in cert-manager?

Reply:
Basically the existence of Certificate and Issues CRs, this will determine that cert-manager.io will be the one filling up certificates in the secret, which will be used by KLT operators :)